config/locales/account_reset/es.yml

```diff
@@ -23,8 +23,9 @@ es:
     delete_account:
       are_you_sure: "¿Seguro que quieres eliminar tu cuenta?"
       info: Eliminar su cuenta debe ser su último recurso si está bloqueado de tu
-        cuenta No podrá recuperar ninguna información vinculada a su cuenta. Una vez
-        que se elimine su cuenta, puede crear una nueva usando la misma dirección
+        cuenta No podrá recuperar ninguna información vinculada a su cuenta. Notificaremos
```

Comment:
Can you re-check with your team on Spanish copy?
It makes few sense to my reading. Suggestion

```
Eliminar su cuenta debe ser su último recurso si su cuenta está bloqueada.
No podrá recuperar ninguna información vinculada a su cuenta. Notificaremos
a las agencias a las que acceda con %{app} que ya no tiene una cuenta.
Cuando su cuenta sea eliminada podrá crear una nueva usando la
misma dirección de correo electrónico.
```

I'm not a professional translator, but a native Spanish speaker (es-mx variation).

Reply:
Thanks!